ios/Podfile

```diff
@@ -7,7 +7,6 @@ install! 'cocoapods', :deterministic_uuids => false
 target 'NewExpensify' do
   permissions_path = '../node_modules/react-native-permissions/ios'
 
-  pod 'Plaid', '~> 2.3.1'
```

Comment:
Is this auto-linked in a newer version?

Reply:
Yes correct. I don't know the history behind why we added this, but I think we should rely on auto linking all pod libraries unless there is a specific version breaking for us.

Reply:
I don't understand their documentation.. It says it uses auto linking, so we shouldn't need to add it to the Podfile manually. 

Plaid 2.5.1 is installed via autolinking and can be seen in the Podfile.lock: https://github.com/Expensify/App/pull/10439/files#diff-b7d85c35701918f25c9b2c418249f9b66fd9b067676e295a9eff493b73ee1752R231

And the Plaid SDK seems to boot just fine:
<img width="564" alt="Screen Shot 2022-08-19 at 12 30 39 PM" src="https://user-images.githubusercontent.com/2838819/185684224-46299bb1-d4ed-4bbb-9ce4-a7aeeaf94363.png">
<img width="564" alt="Screen Shot 2022-08-19 at 12 31 34 PM" src="https://user-images.githubusercontent.com/2838819/185684318-467b13a8-87bd-4ac2-ba53-ce4a9652e410.png">